src/scale.c

```diff
@@ -4,25 +4,10 @@
 #include "avif/internal.h"
 
 #if !defined(AVIF_LIBYUV_ENABLED)
-
-avifBool avifImageScale(avifImage * image,
-                        uint32_t dstWidth,
-                        uint32_t dstHeight,
-                        uint32_t imageSizeLimit,
-                        uint32_t imageDimensionLimit,
-                        avifDiagnostics * diag)
-{
-    (void)image;
-    (void)dstWidth;
-    (void)dstHeight;
-    (void)imageSizeLimit;
-    (void)imageDimensionLimit;
-    avifDiagnosticsPrintf(diag, "avifImageScale() called, but is unimplemented without libyuv!");
-    return AVIF_FALSE;
-}
-
+#include "scale/avif_scale.h"
```

Comment:
Rather than relying on a custom header file, could we use `extern`-imported functions here? This question is only relevant if no original libyuv header containing `ScalePlane()`/`ScalePlane_12()` definitions is imported.